iib/web/api_v1.py

```diff
@@ -513,7 +538,9 @@ def add_bundles() -> Tuple[flask.Response, int]:
     messaging.send_message_for_state_change(request, new_batch_msg=True)
 
     overwrite_from_index = payload.get('overwrite_from_index', False)
-    celery_queue = _get_user_queue(serial=overwrite_from_index)
+    celery_queue = _get_user_queue(
+        serial=overwrite_from_index, from_index_pull_spec=request.from_index.pull_specification
```

Comment:
What would happen if the request.from_index.pull_specification would not be specified?

Reply:
Nice catch, please review.